### ISSUE TITLE
[QUESTION] All about these indexes?

### ISSUE THREAD

**Issue by Ph0rk0z** [gitmylo/audio-webui#45]
So what is the deal on this? It has to now be shared with the models? The files are gigantic. I notice that my training loss is looking lower and the models sound a tiny bit nicer. Singing is way better. No more noise on empty sections of audio either.

See a lot of up and down on the graph after a certain number of epochs. The scale is really tiny so I can't tell much besides it zig-zagging towards 0. No idea on final loss unless it's in a text file somewhere. Would going for 200 epochs help now? Previously I didn't get much out of it as loss would flatline.

**Comment by gitmylo** [comment]
Index files are optional, they do not affect training. They contain basically all features from your training data. I believe a similarity search is used on them in order to fit accents and speech patterns better.

Nothing has changed with training, index is not used during training at all. Not sure why you're seeing differences.

**Comment by Ph0rk0z** [comment]
The huge amount of epochs now? It used to be like 2 or 3.. I ran it for about 20 or 30 before to see what "more" would do and it would flatline and sound more electronic. Now it doesn't crack but it's taking quite a while.. more like any other model. Even in my huge 45 sized batches.

I need to try the models with and without the index to check the difference. Perhaps that is indeed the secret sauce. Definitely will re-extract my downloaded ones and pull out those index files if that's the case.

I notice one of the created indexes is never copied or used and other people's models came with both.

I'm not trying to be a huge pain, but RVC info seems rather limited and the repo is in I think Chinese.

**Comment by gitmylo** [comment]
that might have been something else, not the index though.

i'm trying to figure out as much as i can about RVC.

**Comment by Ph0rk0z** [comment]
Yes, docs on audio models and training in general aren't great. A lot of trying things blind and hoping for the best.
